packages/plugin-react-native-remote-spans/android/src/main/java/com/bugsnag/reactnative/performance/remotespans/BugsnagRemoteSpans.java

```diff
@@ -36,12 +35,14 @@ public class BugsnagRemoteSpans extends NativeBugsnagRemoteSpansSpec {
 
     static final String MODULE_NAME = "BugsnagRemoteSpans";
 
+    private final ReactApplicationContext reactContext;
+
     public BugsnagRemoteSpans(ReactApplicationContext reactContext) {
-        super(reactContext);
+        this.reactContext = reactContext;
     }
 
-    @Override
-    public @Nullable WritableMap getSpanIdByName(String spanName) {
+    @Nullable
+    public WritableMap getSpanIdByName(String spanName) {
         NativeSpanAccessPlugin nativeSpanAccessPlugin = NativeSpanAccessPlugin.getInstance();
```

Comment:
We should probably rename this to `BugsnagRemoteSpansPlugin` for consistency with iOS and JS

Reply:
Now that you say that, I think maybe the `Remote` plugins should be renamed - since they could imply client/server relationship rather than a clear native/js relationship?